concourse/pipelines/artifact-releaser-autopush.yaml

```diff
@@ -157,6 +157,21 @@ jobs:
           publish_version: ((.:version))
           source_version: "v20211027"
           release_notes: "Disregard this release. Alma Linux 8 test."
+      - task: publish-prod-debian-11
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: google/cloud-sdk
+              tag: alpine
+          run:
+            path: /bin/sh
+            args:
+            - -exc
+            - |
+              wf="{\"Name\": \"debian-prod-test\", {{$work_project := printf \"%q\" \"gce-image-builder\" -}} {{$endpoint := \`\"https://www.googleapis.com/compute/alpha/projects/\"\` -}} {{$delete_after := \`\"24h*30*6\"\` -}} {{if eq .environment \"prod\" -}} \"WorkProject\": {{$work_project}}, \"PublishProject\": \"bct-prod-images\", \"ComputeEndpoint\": {{$endpoint}}, \"DeleteAfter\": {{$delete_after}}, {{- else -}} \"WorkProject\": {{$work_project}}, \"PublishProject\": {{$work_project}}, \"ComputeEndpoint\": {{$endpoint}}, \"DeleteAfter\": {{$delete_after}}, {{- end}} {{$time := trimPrefix .publish_version \"v\"}} \"Images\": [{ \"Prefix\": \"debian-11-bullseye\", \"Family\": \"debian-11-prod-test\", \"Description\": \"Debian, Debian GNU/Linux, 11 (bullseye), amd64 built on {{$time}}, supports Shielded VM features\", \"Licenses\": [ {{if (or (eq .environment \"staging\") (eq .environment \"oslogin-staging\")) -}} \"projects/bct-staging-functional/global/licenses/debian-11-bullseye\" {{- else -}} \"projects/debian-cloud/global/licenses/debian-11-bullseye\" {{- end}} ], \"GuestOsFeatures\": [\"UEFI_COMPATIBLE\", \"VIRTIO_SCSI_MULTIQUEUE\"]}]}"
```

Comment:
can we extract this to a file? this is v difficult to read, haha

Reply:
I wanted the workflow saved here to signal that this workflow should not be used anywhere else. Which is why the entire workflow is saved here instead of being read in from `compute-image-tools/daisy_workflows`.

There wasn't an existing directory for saving a file to `guest-test-infra/concourse` so I thought it was best to include directly in the task instead of creating a directory with a single file. If this is saved to a file I don't see a natural place for it, do you have any suggestions?

Reply:
Talked offline, and decided to save the workflow file to `compute-image-tools`.

https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1923